config/sunspot.yml

```diff
@@ -1,24 +1,21 @@
+---
 production:
   solr:
     hostname: localhost
     port: 8983
     log_level: WARNING
-    path: /solr/default
```

Comment:
no, these changes are perfect -- comments not required. Just mentioning how they got added for future reference. Thanks!